db/schema.sql

```diff
@@ -0,0 +1,103 @@
+SELECT count(*) FROM information_schema.tables WHERE table_schema = CURRENT_SCHEMA() AND table_name = 'migrations' AND table_type = 'BASE TABLE';
+
+CREATE TABLE "migrations" (
+    "id" varchar(255),
+    PRIMARY KEY ("id")
+);
+
+CREATE TABLE "tenants" (
+    "created_at" timestamptz NOT NULL,
+    "updated_at" timestamptz,
+    "deleted_at" timestamptz,
```

Comment:
I didn't raise this previously because I didn't realize this is how gorm does things. I strongly recommend you remove the gorm-aut-type-date-things, and use your own, with RDBM-side triggers.

I.e.
```
created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP(),
created_at TIMESTAMPTZ NOT NULL, // + a trigger that does the updates for you at all times 
deleted_at TIMESTAMPTZ
```

Not a blocker-blocker, but you are entering a world of pain if you are relying of various parts of the app to set everything correctly at all times. Triggers just work.